operations/mimir/config.libsonnet

```
@@ -89,6 +89,11 @@
     query_tee_backend_preferred: '',
     query_tee_node_port: null,
 
+    usageStatsConfig:: {
+      'usage-stats.enabled': true,
+      'usage-stats.installation-mode': 'jsonnet',
+    },
```

Nit: I think there's no need to provide `usage-stats.enabled` since it's now enabled by default and only clutters the rendered CLI flags list. I'd remove it as we're still rendering `usage-stats.installation-mode` which gives enough visibility about the feature to whoever is concerned.

OTOH, it should be easy to disable the feature and remove all the related flags, like this:

```suggestion
    // usage_stats_enabled enables the reporting of anonymous usage statistics about the Mimir installation.
    // For more details about usage statistics, see:
    // https://grafana.com/docs/mimir/latest/operators-guide/configure/about-anonymous-usage-statistics-reporting/
    usage_stats_enabled: true,
    usageStatsConfig: if !$._config.usage_stats_enabled then {} else {
      'usage-stats.installation-mode': 'jsonnet',
    },
```

---

👍 Agree with Oleg

---

Graet idea! Thanks!